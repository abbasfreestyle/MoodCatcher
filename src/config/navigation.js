import React from 'react';

import {
  SafeAreaView,
  createAppContainer,
  createStackNavigator
} from 'react-navigation';

// import { setTopLevelNavigator } from 'utils/navigationService';

import HomeScreen from 'screens/Home';

const Screens = createStackNavigator(
  {
    Home: { screen: HomeScreen }
  },
  {
    initialRouteName: 'Home',
    headerMode: 'none'
  }
);

const AppContainer = createAppContainer(Screens);

const App = () => {
  return (
    <SafeAreaView flex={1} backgroundColor="#EEE">
      <AppContainer
      // ref={navigatorRef => {
      //   setTopLevelNavigator(navigatorRef);
      // }}
      />
    </SafeAreaView>
  );
};

export default App;
