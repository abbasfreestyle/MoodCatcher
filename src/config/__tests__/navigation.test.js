import { screensConfig } from '../navigation';

describe('Navigation.', () => {
  describe('Snapshots.', () => {
    test('should match', () => {
      expect(screensConfig).toMatchSnapshot();
    });
  });
});
