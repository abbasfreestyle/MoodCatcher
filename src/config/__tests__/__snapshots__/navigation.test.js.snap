// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation. Snapshots. should match 1`] = `
Array [
  Object {
    "AddComment": Object {
      "screen": [Function],
    },
    "Dashboard": Object {
      "screen": [Function],
    },
    "Home": Object {
      "screen": [Function],
    },
    "SelectMood": Object {
      "screen": [Function],
    },
    "Thanks": Object {
      "screen": [Function],
    },
  },
  Object {
    "defaultNavigationOptions": Object {
      "headerStyle": Object {
        "backgroundColor": "#A073B2",
      },
      "headerTintColor": "#FFFFFF",
    },
    "initialRouteName": "Home",
  },
]
`;
